lemur/certificates/views.py

```diff
@@ -1456,6 +1462,7 @@ def __init__(self):
         super(CertificateExport, self).__init__()
 
     @validate_schema(certificate_export_input_schema, None)
+    @operator_permission.require(http_exception=403)
```

Comment:
Post is deceptive here, an export is functionally a get of a cert that's reformatted